task-json/q2.js

```diff
@@ -0,0 +1,20 @@
+
+const timeinseconds = [4,5,2,10,0];
+function delay (seconds){
+    return new  Promise(function(resolve, reject) {
+        
+       setTimeout(resolve(),seconds*1000);
+        
+
+
+     
+    }).then(function() {
+       
+         console.log(`${seconds} seconds` );
+        
+       
+    });
+}
+for(let x=0 ;x<timeinseconds.length;x++){
+    delay(timeinseconds[x]);
+}
```

Comment:
Did you test the results?

Reply:
yes its not working as it should be its printing the numbers in the same order  in the array but the timeout function is not responding